### ISSUE TITLE
update disconnect method for opsdroid 0.13.0

### ISSUE THREAD

**Issue by colonelkrud** [opsdroid/connector-matrix#37]
Latest opsdroid gives the following exception when receiving a stop signal:
```INFO opsdroid.core: Stopping connector ConnectorMatrix...
/home/opsdroid/.local/share/opsdroid/opsdroid-modules/connector/matrix/connector.py:201: RuntimeWarning: coroutine 'ClientSession.close' was never awaited
  self.session.close()
INFO opsdroid.core: Stopped connector ConnectorMatrix
INFO opsdroid.core: Stopping database ...
ERROR opsdroid.core: Caught exception
ERROR opsdroid.core: {'message': 'Task exception was never retrieved', 'exception': TypeError('disconnect() takes 1 positional argument but 2 were given',), 'future': <Task finished coro=<OpsDroid.handle_signal() done, defined at /usr/local/lib/python3.6/dist-packages/opsdroid/core.py:120> exception=TypeError('disconnect() takes 1 positional argument but 2 were given',)>}
```

**Comment by jacobtomlinson** [comment]
After thinking about this I'm not sure the `Connector` API is quite right. There is no compelling reason why an opsdroid pointer should have to be passed on disconnect as the connector should already have a pointer to it stored as an attribute from when it was set up.

See opsdroid/opsdroid#725 for more information on my proposed change.

**Comment by jacobtomlinson** [comment]
As the change will not be in until the next release you may wish to make the change but ensure it is an optional kwarg so that when the API changes again it will not break. You could then remove this when matrix is merged as a core connector.

**Comment by Cadair** [comment]
I fixed this in #36

**Comment by Cadair** [comment]
This can be closed.
